handlers/transcode_test.go

```diff
@@ -0,0 +1,162 @@
+//go:build externaldeps
+// +build externaldeps
+
+/* Depends:
```

Comment:
I think there's too much environmental dependency here for it to be useful as a unit test. Since you've tested and are happy the new functionality works, can we get rid of this / break into down into unit tests with e.g a fake Mist process